_posts/2021-04-08-cloud-platform.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+img: assets/images/articles/cloud-platform.jpg
+title: "AWS vs. Azure vs. Google Cloud: Which Cloud Platform to Choose?"
+description: "When it comes to cloud solutions, Google, Microsoft, and Amazon are the frontrunners. This is because they provide the most secure, versatile, and reliable cloud services to clients and businesses worldwide. They also offer a wide range of storage, computing, and networking services, among many others. This is where the big three cloud platforms come into the picture—Google Cloud Platform (GCP), Microsoft Azure, and Amazon Web Services (AWS)."
+---
+
```

Comment:
@SebastianMera, the word doc didn't had the title, but to be in sync with the rest of the blog posts let's add the title here as well as a heading 1, 
e.g. 

`# AWS vs. Azure vs. Google Cloud: Which Cloud Platform to Choose?`